appveyor.yml

```diff
@@ -34,12 +34,12 @@ test_script:
   - npm run eslint
   - npm run test-cov
 
-after_script:
+after_test:
   - npm install coveralls
-  - cat ./coverage/lcov.info | ./node_modules/coveralls/bin/coveralls.js
-
+  - node ./node_modules/coveralls/bin/coveralls.js < ./coverage/lcov.info
```

Comment:
I think appveyor shouldn't run coveralls, otherwise duplicate results can be confusing.

Reply:
I dropped it (commented, just in case)